.circleci/config.yml

```diff
@@ -219,3 +219,16 @@ jobs:
             ssh-add ~/.ssh/id_rsa_* ~/.ssh/id_rsa
           fi
           ./script/chart_sync.sh kubernetes-bitnami kubernetes@bitnami.com
+  push_images:
+    machine: true
+    steps:
+      - <<: *exports
+      - run: |
+          if [[ -z "$CIRCLE_PULL_REQUEST" && -n "$DOCKER_USERNAME" && -n "$DOCKER_PASSWORD" ]]; then
+            docker login -u="${DOCKER_USERNAME}" -p="${DOCKER_PASSWORD}"
+            for IMAGE in kubeapps/chart-repo kubeapps/chartsvc kubeapps/apprepository-controller kubeapps/dashboard kubeapps/tiller-proxy; do
+              docker pull $IMAGE:$DEV_TAG
+              docker tag $IMAGE:$DEV_TAG $IMAGE:$PROD_TAG
```

Comment:
BTW, since these tags are public, we might want to `untag` the temporary build tags once the promotion is finished? Don't know if that's the right thing to do just because of noise. WDYT? cc/ @prydonius @andresmgot  

![selection_676](https://user-images.githubusercontent.com/24523/45061213-5dac8c80-b058-11e8-85f6-ec396bdfbd7c.png)

Reply:
> I'd rather not delete tags, they may be useful to reproduce a build

This can be achieved referencing the image via the @sha256 no? We could output it in the logs and in case you want to pull it exactly you just reference the sha.

> also it's a bit risky in cause something unexpected happens

What do you mean, anything in mind?

I think that probably we should not worry about this for now since that image is not useful by itself, although I'd rename the `build-abc` tag for something like `ci-build-xx`.

Reply:
> This can be achieved referencing the image via the @sha256 no? 

I don't know if deleting the tag would cause that the build is not reachable even using the sha256. I would expect that docker hub garbage collector removes it.

>> also it's a bit risky in cause something unexpected happens

> What do you mean, anything in mind?

Yes, I was thinking on deleting `latest` or a versioned tag by accident (although that would be improbable).

Let me use a different image name, something like `kubeapps/tiller-proxy-ci:build-ABC`. I think that's a neat solution that avoid polluting the production image with tags and also prevents from unexpected deletions.

Reply:
> Let me use a different image name, something like kubeapps/tiller-proxy-ci:build-ABC. I think that's a neat solution that avoid polluting the production image with tags and also prevents from unexpected deletions.

Well, I am not sure if that's better since the image will appear in search results and so on. In any case I think that it does not really matter. Choose whatever solution you think it's best. Thanks!